stream-product/product-core/src/main/java/com/backbase/stream/product/mapping/ProductMapper.java

```diff
@@ -104,6 +104,7 @@ public interface ProductMapper {
     @Mapping(source = ProductMapperConstants.EXTERNAL_ID, target = ProductMapperConstants.EXTERNAL_ARRANGEMENT_ID)
     @Mapping(source = ProductMapperConstants.PRODUCT_TYPE_EXTERNAL_ID, target = ProductMapperConstants.EXTERNAL_PRODUCT_ID)
     @Mapping(source = ProductMapperConstants.LEGAL_ENTITIES, target = ProductMapperConstants.EXTERNAL_LEGAL_ENTITY_IDS)
+    @Mapping(source = "currentInvestment", target = "currentInvestmentValue")
```

Comment:
Wouldn't this solve the problem instead of creating a specific method?
`@Mapping(source = "currentInvestment.amount", target = "currentInvestmentValue")`

Reply:
Yes, that is much better approach. Thanks!